epilepsy12/general_functions/email.py

```diff
@@ -0,0 +1,18 @@
+# Django Imports
+from django.utils.html import strip_tags
+from django.core.mail import send_mail
+from django.conf import settings
+
+
+def send_email_to_recipients(recipients: list, subject: str, message: str):
+    """
+    Sends emails
+    """
+    send_mail(
```

Comment:
After discussing during in person code review we decided to remove the try/catch that was in the old version of this function as it would mask issues if they were to occur in production and we are confident that sensitive error data won't be revealed to users in production